oop/games/instrument.py

```diff
@@ -0,0 +1,40 @@
+from oop.games.rhythm import Rythym
+# instrument representing games with components that are instruments or represent instruments
```

Comment:
good comments - recall that docstring comments use triple quotes (""") and go *under* the class definition.